http-api/src/main/java/io/avaje/http/api/PathTypeConversion.java

```diff
@@ -22,6 +28,10 @@ public static String withDefault(String value, String defaultValue) {
     return value != null ? value : defaultValue;
   }
 
+  public static List<String> withDefault(List<String> value, String defaultValue) {
```

Comment:
Just allow us to define a `List<String> defaultValue` parameter instead? Some defaults may include multiple entries.

Reply:
ok